backend/data/seed.js

```diff
@@ -26,9 +29,16 @@ async function seedConfigs() {
 // Seed function
 export async function seedDatabase() {
   try {
-    // only connect if not testing otherwise connect to test db
-    let mongoURI = process.env.MONGO_URI + "/cdj"; // pulled from compose.yaml env vars
-    if (process.env.NODE_ENV === 'test') { mongoURI = mongoURI + '-test'; }
+    // Set the URI for the database
+    let mongoURI = process.env.MONGO_URI + "/cdj";
+
+    // If testing, use the in-memory database
+    if (memoryMongo) { 
+      console.log("Setting up in-memory database...")
+      await memoryMongo.start();
+      mongoURI = memoryMongo.getUri();
+     }
```

Comment:
That's embarrassing 😅 Nice catch! I kept telling myself I'd set up the eslint rules so that saving in VS Code will automatically apply them and lint the code. It's been a while since I worked on the backend and forgot to run the linter manually.